### PR TITLE
Update polyphone to 1.9

### PR DESCRIPTION
**Pull request by gtmgianni**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by commitay** [comment]
Download URL 404s.

The download from the homepage is:

> https://polyphone-soundfonts.com/en/files/file/559-polyphone-1-9-dmg/latest/download?0fdc953d1ce17e5d6f055f0d6ef83577=1&return=aHR0cHMlM0ElMkYlMkZwb2x5cGhvbmUtc291bmRmb250cy5jb20lMkZlbiUyRmZpbGVzJTJGNi1wb2x5cGhvbmUlMkY0MTMtcG9seXBob25lLTEtOQ==

`referer`, etc don't work. Might have to remove this.

File: Casks/polyphone.rb [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
-  version '1.8'
-  sha256 '877f149b19c12c03f4c97b5f06b4c0e9d0e2b84c674a93146619c1cc24b16b69'
+  version '1.9'
+  sha256 '40d6a28e941d7ccc398782154f93290e1f9e3fbe7b9754c394f32e6a92636efa'
 
-  # sourceforge.net/polyphone was verified as official when first introduced to the cask
-  url "https://downloads.sourceforge.net/polyphone/polyphone-#{version}.dmg"
-  appcast 'https://sourceforge.net/projects/polyphone/rss',
-          checkpoint: '609b92b4f61512809cb2bac131873deac0b62d085a2ae617415d2fe585107fc3'
+  url "https://polyphone-soundfonts.com/judownload/0_1512442550_rk9xVE/polyphone-#{version}.dmg"
```
Comment (claui) [inline Casks/polyphone.rb]: Use this URL, which is more stable:

```
url "https://polyphone-soundfonts.com/en/download/file/559-polyphone-#{version.sub('.', '-')}-dmg/latest/download"
```

**Review by claui — CHANGES_REQUESTED** [review]

File: Casks/polyphone.rb (hunk 1 above)
Comment (commitay) [reply Casks/polyphone.rb]: @claui Does that URL work reliably for you?
```
curl: (16) Error in the HTTP2 framing layer
audit for polyphone: failed
 - download not possible: Download failed on Cask 'polyphone' with message: Download failed: https://polyphone-soundfonts.com/en/download/file/559-polyphone-1-9-dmg/latest/download
```

Comment (claui) [reply Casks/polyphone.rb]: @commitay I double-checked one more time; both `install` and `audit --download` work for me.

During installation

After installation/audit

Comment (commitay) [reply Casks/polyphone.rb]: Thanks!

Comment (claui) [reply Casks/polyphone.rb]: @commitay Are you sure the `polyphone.app` artifact is supposed to be in lower-case?

File: Casks/polyphone.rb [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
-  # sourceforge.net/polyphone was verified as official when first introduced to the cask
-  url "https://downloads.sourceforge.net/polyphone/polyphone-#{version}.dmg"
-  appcast 'https://sourceforge.net/projects/polyphone/rss',
-          checkpoint: '609b92b4f61512809cb2bac131873deac0b62d085a2ae617415d2fe585107fc3'
+  url "https://polyphone-soundfonts.com/en/download/file/559-polyphone-#{version.dots_to_hyphens}-dmg/latest/download"
   name 'Polyphone'
   homepage 'https://polyphone-soundfonts.com/'
 
-  app 'Polyphone.app'
+  app 'polyphone.app'
```
Comment (claui) [inline Casks/polyphone.rb]: @commitay Are you sure the `polyphone.app` artifact is supposed to be in lower-case?

Comment (claui) [reply Casks/polyphone.rb]: Mind = blown :woman_shrugging:

Fair enough.

**Review by claui — APPROVED** [review]

**Pull request merged**